app/frontend/src/features/connections/friends/useFriendList.test.tsx

```diff
@@ -86,39 +84,31 @@ describe("when the listFriends query succeeds", () => {
     getUserMock.mockImplementation((userId: string) => {
       return userId === "2"
         ? getUser(userId)
-        : new Promise((r) => setTimeout(() => r(getUser(userId)), 1));
+        : new Promise((r) => setTimeout(() => r(getUser(userId)), 50));
```

Comment:
```suggestion
        : new Promise(() => null);
```

Would that make a never-resolving promise, or would it just throw?

Reply:
I thought this might cause some weird timeout from jest but it actually doesn't, think I'll change it to this when I get some time later so I don't need to specify a time for `waitForNextUpdate` to wait. It seems to know to wait for only resolved promises and not pending ones so will make the test a bit clearer than randomly waiting for 10ms as a hack.